docs/source/storeconv.md

```diff
@@ -92,14 +96,14 @@ interface for access to the available datasets. Note that, under this system:
 
 To work around some of the restrictions of point (2) above, a dataset MAY be
 presented by the opener as multiple "virtual" datasets with different
-parameter schemas. For instance, the hypothetical dataset described above MAY
-be offered not as a single dataset `envdata` but as two datasets
+parameter schemas. For instance, the hypothetical dataset described above MAY 
+be offered as two datasets
 `envdata:sensor-a` (with a fixed *sensor_type* of A) and `envdata:sensor-b`,
 (with a fixed *sensor_type* of B), offering different sets of permitted
-variables.
+variables. However, it may not be represented as a single dataset `envdata`.
```

Comment:
```suggestion
variables.
```

No, that's not what I meant! It's also allowed to offer a single dataset. Probably best to remove this clause entirely to avoid misunderstanding.